starter/source/output/th/hm_read_thgrou.F

```diff
@@ -2736,11 +2742,15 @@ SUBROUTINE HM_READ_THGROU(
       !-------------------------------------------
       ! /TH/SUBS
       !-------------------------------------------
+      !//TODO: FOR SUBS 
```

Comment:
you have several 'TODO' , can you remove , as I think you already implemented this ?